test/e2e/atlas/access_lists_test.go

```diff
@@ -53,7 +53,7 @@ func TestAccessList(t *testing.T) {
 			g.projectID,
 			"-o=json")
 		cmd.Env = os.Environ()
-		resp, err := cmd.CombinedOutput()
+		resp, err := e2e.RunAndGetStdOut(cmd)
```

Comment:
Should we check that the `os.Stderr` is empty in all the tests? I am wondering if checking for `require.NoError(t, err)` (next line) may not be enough since the command errors will be shown only in `Stderr`

Reply:
`os.Stderr` is used to print our logs; it doesn't mean an error has occurred.

Some places do use `os.Stderr` to check if some log has been printed; I've checked those places manually and made sure that the test still works.